src/main/java/org/netcracker/educationcenter/elasticsearch/database/operations/FTPFileObjectOperations.java

```diff
@@ -0,0 +1,74 @@
+package org.netcracker.educationcenter.elasticsearch.database.operations;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.netcracker.educationcenter.elasticsearch.Connection;
+import org.netcracker.educationcenter.elasticsearch.database.model.FTPFileObject;
+
+import java.io.IOException;
+
+/**
+ * This class implements Elasticsearch Database operations with (on) FTP file objects
+ *
+ * @author Mikhail Savin
+ * @see ElasticsearchOperations
+ */
+public class FTPFileObjectOperations extends ElasticsearchOperations {
+
+    /**
+     * FTP file object model's index
+     */
+    private static final String INDEX = "ftpfileobjects";
+
+    /**
+     * Creates a new FTPFileObjectOperations instance with given connection to interact with ES DB.
+     *
+     * @param connection current connection
+     */
+    public FTPFileObjectOperations(Connection connection) {
+        super(connection);
+    }
+
+    /**
+     * Inserts given FTP file object into the ES Database
+     *
+     * @param ftpFileObject FTP file object to insert
+     */
+    public void insertFTPFileObject(FTPFileObject ftpFileObject) throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
ObjectMapper can be field of parent class

Reply:
ObjectMapper moved to parent class. Subclasses access it with the getMapper() method.